templates/api-blueprint.swig

```diff
@@ -0,0 +1,105 @@
+FORMAT: 1A
+{% if api.meta.HOST %}HOST: {{ api.meta.HOST }}{% endif %}
+
+
+
+
+
+{% macro renderExample(example) %}
+  {% if example.headers && example.headers.length > 1 %}
+    + Headers
+
+      {% for header in example.headers.content %}
+        {% if header.meta.name.toLowerCase() != 'content-type' %}
+        + {{ header.meta.name }}: {{ header.content }}
+        {% endif %}
+      {% endfor %}
+
+  {% endif %}
+  {% if example.messageBody %}
+    + Body
+
+            {{ example.messageBody.content|indent(12) }}
+
+  {% endif %}
```

Comment:
It would be nice if when there wasn't any headers or other sections, we don't use `+ Body`. Like https://github.com/apiaryio/Paw-APIBlueprintGenerator/blob/master/apiblueprint.mustache#L14-18